server/services/interventionsService.test.ts

```diff
@@ -2910,6 +2911,59 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     })
   })
   */
+
+  describe('getServiceProviderReportingData', () => {
+    const reportingResponse = [
+      {
+        referralLink: 'https://refer-and-monitor.example/referral/c9f9e22f-ddd9-422a-a9af-76fc02b18b38',
+        referralRef: 'SM1973AC',
+        referralId: 'c9f9e22f-ddd9-422a-a9af-76fc02b18b38',
+        contractId: 'thing',
+        organisationId: 'XYZ5678',
+        caseworkerId: 'liane.supplier@example.com',
```

Comment:
Also a uuid?

Reply:
Looking at the v1.0.0 schema on the Excel file on [IC-1116](https://dsdmoj.atlassian.net/browse/IC-1116), the example value is `liane.supplier@example.com`.